broadcast/src/linked/safe/mod.rs

```diff
@@ -0,0 +1,14 @@
+//! Provides a wrapped version of the `wire` module.
+//! This allows for more robust error handling and type checking.
+
+use commonware_cryptography::Scheme;
+
+// Exports certain types from the private module.
+mod private;
+pub use private::{Ack, Chunk, Parent};
+
+/// Safe version of a `Link`.
+///
+/// This type is exported publicly for simplicity, while allowing the sealed representation to use
```

Comment:
I think we should just implement `Debug` and `PartialEq` rather than doing a number of extra exports.

Here is how:
* https://github.com/commonwarexyz/monorepo/blob/9f67390d2757dbc7a9f7ddddafd8a660e1dfbb9d/cryptography/src/bls12381/scheme.rs#L163-L173
* https://github.com/commonwarexyz/monorepo/blob/9f67390d2757dbc7a9f7ddddafd8a660e1dfbb9d/cryptography/src/ed25519/scheme.rs#L134-L138